code/obj/stool.dm

```diff
@@ -874,6 +876,23 @@
 			AM.changeStatus("weakened", 2 SECONDS)
 		return
 
+/* ======================================================= */
+/* -------------------- Material Chairs ------------------ */
+/* ======================================================= */
+
+/obj/stool/chair/material
+	name = "material chair"
+	desc = "A chair made from a material"
+	uses_material_appearance = TRUE
+	mat_changename = TRUE
+
+/obj/stool/chair/material/mauxite
+	name = "chair"
+	icon_state = "chair$$mauxite"
```

Comment:
hmm Is this necessary, I reckon the setMaterial code will replace it with this icon state automatically

Reply:
for map editor sprites